src/test/regress/expected/multi_copy_with_ltree.out

```diff
@@ -0,0 +1,14 @@
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1430000;
```

Comment:
I think we do not need to new test file for this. You can add your tests at multi_copy.source